Source/ThirdParty/jsep.js

```diff
@@ -680,5 +680,9 @@ define(function() {
 	}
 }(this));
 
-    return jsep.noConflict();
+    if (typeof jsep.noConflict === 'function') {
```

Comment:
Change this to `if (typeof jsep !== 'undefined') {` and remove the last return statement on line 687.  I think that will finally do it.  Add a comment to the if that mentions that `jsep` only exists when running in a browser, not Node.